pyogrio/tests/test_geopandas_io.py

```diff
@@ -1062,6 +1063,21 @@ def test_write_empty_dataframe(tmp_path, ext, use_arrow):
     assert_geodataframe_equal(df, expected)
 
 
+def test_write_empty_geometry(tmp_path):
+    expected = gp.GeoDataFrame({"x": [0]}, geometry=from_wkt(["POINT EMPTY"]), crs=4326)
+    filename = tmp_path / "test.shp"
```

Comment:
```suggestion
    filename = tmp_path / "test.gpkg"
```

per [comment in #436](https://github.com/geopandas/pyogrio/issues/436#issuecomment-2196478022), using GPKG sidesteps the issue below, and you can enable lines 1077-1078